app/Providers/AppServiceProvider.php

```
@@ -22,6 +23,6 @@ public function register(): void
      */
     public function boot(): void
     {
-        //
+        Model::shouldBeStrict(! app()->isProduction());
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\Illuminate\Database\Eloquent\Model' in method 'boot'. (undefined)

(StaticAccess)